src/parser/bison_parser.y

```diff
@@ -809,7 +830,7 @@ opt_having:
 	;
 
 opt_order:
-		ORDER BY order_list { $$ = $3; }
+		ORDER BY order_list { $$ = $3; };
```

Comment:
Why? The diff above doesn‘t seem to have this either.

Reply:
True, semicolon seems to have sneaked in there